recipe/build.sh

```diff
@@ -5,7 +5,9 @@ module_path="${GOPATH:-"$( go env GOPATH )"}"/src/github.com/containers/skopeo
 mkdir -p "$( dirname "${module_path}" )"
 mv ./src "${module_path}"
 
-export GO111MODULE=off
+
+go mod init "${module_path}"
+go mod tidy
```

Comment:
I think we can remove these two lines. Because

`go mod init` is help to generate `go.mod/go.sum` (like a index)
`go mod tidy` is help to donwload the go module deps. (real deps)

We can see [go.mod](https://github.com/containers/skopeo/blob/main/go.mod) already exists in skopeo and :
https://github.com/containers/skopeo/blob/0c43e43e364134ba28d7c0309a0a4ff819aa7fc9/Makefile#L43-L46

(The skopeo use the vendor to manage the module deps)